src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1452,6 +1508,44 @@ private void PlaceIfPossible(string organelleType, int q, int r, int rotation, r
         }
     }
 
+    /// <summary>
+    ///   A recursive function that adds my neighbors to the checked list and calls the function for each neighbor.
+    /// </summary>
+    /// <param name="checked">
+    ///   The list of visited hexes will be filled up.
+    /// </param>
+    /// <param name="me">
+    ///   Which neighbors to visit next.
+    /// </param>
+    private void CheckmarkNeighbors(List<Hex> @checked, Hex me)
+    {
+        // Get all neighbors not already visited
+        var myNeighbors = GetNeighborHexes(me).Where(p => !@checked.Contains(p)).ToArray();
+
+        // Add the new neighbors to the list
+        @checked.AddRange(myNeighbors);
+        foreach (var neighbor in myNeighbors)
```

Comment:
Add a blank line before this foreach as it doesn't seem to logically relate to the previous line and the comment.